base-theme/assets/webpack/webpack.common.ts

```diff
@@ -78,40 +82,6 @@ const config: webpack.Configuration = {
           "postcss-loader",
           "sass-loader"
         ]
-      },
-
-      {
-        test: require.resolve("shifty"),
-        use:  [
-          { loader: "expose-loader", options: { exposes: ["NGTweenable"] } }
-        ]
-      },
-
-      {
-        test: require.resolve("hammerjs"),
-        use:  [
-          {
-            loader:  "expose-loader",
-            options: { exposes: ["NGHammer"] }
```

Comment:
In our previous workaround (the code being removed here), nanogallery2 was actually assigning NGHammer to `window`, and then this re-assigns it. With some other webpack changes I'm working on, webpack complained about this **re-assignment**. Which is the immediate motivation for this PR, in addition to removing unnecessary deps.

_In contrast, Nanogallery2 was not assigning NGTweenable globally, which it is supposed to do._